common/test/acceptance/pages/studio/users.py

```diff
@@ -35,7 +35,7 @@ def is_browser_on_page(self):
         """
         Returns True iff the browser has loaded the page.
         """
-        return self.q(css='body.view-team').present
+        return self.q(css='ol.user-list').visible
```

Comment:
I'm not sure this will fix the issue...

The page's initial HTML is:
```html
      <ol class="user-list" id="user-list">
        <div class="ui-loading">
          <p><span class="spin"><i class="icon fa fa-refresh"></i></span> <span class="copy">Loading</span></p>
        </div>
      </ol>
```
If I disable JavaScript then load the page, I can definitely still see that OL there. 

Perhaps `self.q(css='ol.user-list').visible and not self.q(css='ol.user-list .ui-loading').visible` ?